### ISSUE TITLE
Site search: Add a search function to the site.

### ISSUE THREAD

**Issue by dbazhenov** [percona/community#318]
It is necessary to add search functionality to the site. 
Search by events, blog articles, and pages.

We use the Hugo engine and it offers the following options
https://gohugo.io/tools/search/

You can use any other solution. It can be a JS widget without a separate page or with a separate search results page.

**Comment by saimaheshtaduri** [comment]
I can work on this as part of hacktoberfest, can you assign to me ?

**Comment by Hermes179** [comment]
Hey @dbazhenov, I'd like to give it a go, Can you assign this issue to me?

**Comment by dbazhenov** [comment]
@Hermes179 Yes, of course you can work on that. Let me know if you have any questions.

**Comment by ImTheKai** [comment]
My first implementation making this work can be found here https://github.com/ImTheKai/community/tree/pagefind

**Comment by ImTheKai** [comment]
Done - https://github.com/percona/community/pull/1041
